lib/db/schema/patch-059-060.sql

```diff
@@ -0,0 +1,232 @@
+CREATE PROCEDURE `createSessionToken_6` (
```

Comment:
Hmm, interesting idea, although I'm somwhat wary of making unconventional changes in this repo.

I'd need to check with @jrgm or @jbuck just to make sure that older versions of stored procedures are always available to call in prod. No reason for thinking they won't be but also no reason for thinking they will be, given that I don't know anything at all about our production environment.